gpos/xml/scap/ssg/content/ssg-chainguard-gpos-ds.xml

```diff
@@ -6021,22 +6124,23 @@
                         administrator and ISSO is one method for mitigating this risk. Such a
                         capability greatly reduces the risk that operating system accessibility will
                         be negatively affected for extended periods of time and also provides
-                        logging that can be used for forensic purposes.
-
-                        To address access requirements, many operating systems can be integrated
-                        with enterprise-level authentication/access/auditing mechanisms that meet or
-                        exceed access control policy requirements.
-
-<html:pre>Script Verification:
-                            To manually verify, run command 'cat /etc/shadow and verify that no
-                            additional users exist</html:pre></ns0:description>
+                        logging that can be used for forensic purposes. To address access
+                        requirements, many operating systems can be integrated with enterprise-level
+                        authentication/access/auditing mechanisms that meet or exceed access control
+                        policy requirements.
+
+                        Script Verification:
```

Comment:
I think the reason our stig had the pre-formatted text was to make it stand out in an oscap html report, like so: 
<img width="1761" height="885" alt="Screenshot from 2025-07-20 21-56-14" src="https://github.com/user-attachments/assets/2ef8cb7b-5f52-4051-a339-9b32c784eb30" />

But I'm not sure this is the proper way to express manual steps like this, or if there's a separate tag instead of `description` that should be used.

Reply:
Yea, I was having all sorts of issues with that. I'll see about fixing that.